src/feeder/et_feeder.cpp

```diff
@@ -173,4 +173,4 @@ void ETFeeder::readNextWindow() {
       dep_free_node_queue_.emplace(node);
     }
   }
-}
+}
```

Comment:
This change is minor, but it's unnecessary. Please avoid introducing unnecessary changes that are unrelated to this PR.

Reply:
It was necessary to pass linter tests with the new linter version.